app/scripts/sagas/index.js

```diff
@@ -128,6 +132,38 @@ export function * fetchPeerIds () {
   }
 }
 
+export function * loadConfig (action) {
+  try {
+    const ipfsConfig = yield call(api.getConfig)
+    const configJSON = JSON.stringify(ipfsConfig, undefined, 2)
```

Comment:
Stringify is really part of the display logic, that should happen only in the component and not in the state I'd say